### ISSUE TITLE
Improve Form Accessibility

### ISSUE THREAD

**Issue by shreyad2806** [Anjaliavv51/Retro#20]
The form lacks proper labels and ARIA attributes, making it difficult for screen readers to interpret. I can add accessible form elements including proper labels, aria-* attributes, and improve focus states for better usability.
Please assign this issue to me under gssoc'24 EXTENDED with label gssoc'24 and hacktoberfest.

**Comment by Anjaliavv51** [comment]
@shreyad2806 i didn't grt u pls add proper screenshot and explain

**Comment by shreyad2806** [comment]
ARIA are used to increase accessibility. These attributes communicate role, state, and property to assistive technologies via accessibility APIs found in modern browsers. Aria-label can be a very powerful tool to use to make sure your site is ADA accessible.
So currently your  ADA score is 68 by adding aria I can make it 100. Your high ADA score really helps when you present your projects in your resume or display it to foriegn universities and recruiters.
![WhatsApp Image 2024-10-02 at 00 09 13_d1bf2f1b](https://github.com/user-attachments/assets/6cfc3534-641e-4985-8650-83795b64f207)

**Comment by Anjaliavv51** [comment]
@shreyad2806 go ahead.
